__tests__/app.test.js

```diff
@@ -49,6 +57,44 @@ describe('CORE: GET /api/topics', () => {
     });
 });
 
+describe('CORE: GET /api', () => {
+    test('returns 200 status code', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+    })
+    test('returns an object', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((response) => {
+            const apiObj = response.body;
+            expect(typeof apiObj).toBe('object');
+        })
+    })
+    test('returns the correct representation (api:description) of all apis within endpoint.json file', () => {    
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((response) => {
+            const response_API_Obj = response.body;
+            return fs.readFile(`${__dirname}/../endpoints.json`,'utf-8')
+            .then((contents) => {
+            const full_API_JSON_Obj = JSON.parse(contents);
+            const expected = {}
+            for(const key in full_API_JSON_Obj){
+                expected[key] = full_API_JSON_Obj[key].description
+            }
+            
+            expect(response_API_Obj).toEqual(expected);
```

Comment:
Not sure that the for in loop is needed here. I think you can just compare the json and the returned object.